athenacli/main.py

```diff
@@ -616,9 +616,10 @@ def is_mutating(status):
 @click.option('--work_group', type=str, help="Amazon Athena workgroup in which query is run, default is primary")
 @click.option('--athenaclirc', default=ATHENACLIRC, type=click.Path(dir_okay=False), help="Location of athenaclirc file.")
 @click.option('--profile', type=str, default='default', help='AWS profile')
+@click.option('--tableformat', type=str, default='csv', help='Table format used with -e option.')
```

Comment:
We are using `table_format` in [athenaclirc](https://github.com/dbcli/athenacli/blob/master/athenacli/athenaclirc#L49). Could we change `--tableformat` to `--table-format`?

Reply:
Of course; updated.